src/rust/grapl-web-ui/src/services/mod.rs

```diff
@@ -36,11 +37,15 @@ pub(self) async fn fwd_request_to_backend_service(
         .request_from(new_url.as_str(), req.head())
         .no_decompress();
 
-    let mut res = forwarded_req.send_stream(payload).await.map_err(|error| {
-        tracing::error!(%error);
+    let mut res = forwarded_req
+        .trace_request()
```

Comment:
This adds a span for the client-side request. This can be helpful when trying to figure out why a request isn't working or reaching a service. This becomes waaaay more useful if we ever need to request a service that we don't control